sympy/sets/sets.py

```diff
@@ -428,7 +428,7 @@ def boundary(self):
         A point x is on the boundary of a set S if
 
         1.  x is in the closure of S.
-            I.e. Every neighborhood of x contains a point in S.
+            I.e. Every neighbourhood of x contains a point in S.
```

Comment:
A neighbourhood (Commonwealth English), or neighborhood (American English)
Courtesy : Wikipedia 
So you being senior in community tell me what should I do ? 
I will revert this line on your words.